modules/osd-aws-privatelink-architecture.adoc

```diff
@@ -1,21 +1,23 @@
 [id="osd-aws-privatelink-architecture.adoc_{context}"]
 = PrivateLink architecture
 
-The Red Hat managed infrastructure responsible for creating PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints. 
+The Red Hat managed infrastructure that creates AWS PrivateLink clusters is hosted on completely private subnets. The only connection made between Red Hat and the customer provided infrastructure is through PrivateLink VPC Endpoints.
 
-The following diagram shows an example of a multiple availability zone (Multi-AZ) PrivateLink cluster deployed only on private subnets. 
+The following diagram shows an example of a multiple availability zone (Multi-AZ) PrivateLink cluster deployed only on private subnets.
 
 .Multi-AZ PrivateLink cluster deployed on private subnets
 
 image::156_OpenShift_ROSA_Arch_0621_privatelink.svg[Multi-AZ PrivateLink cluster deployed on private subnets]
 
-AWS provides multiple reference architectures. Here are three examples:
+== AWS architecture for internet access without an internet gateway
```

Comment:
Consider removing this subhead and providing more context above instead. The ToC above is a bit straining to parse with two level 1 headings and a single level 2 heading that is also quite lengthy

Reply:
I agree the level 2 is jarring, so it's now a level 1. The reason I added a heading is because the information is not directly related to PrivateLink, but is more of a reference for customers who don't know how to set up internet access without using an internet gateway. The original version was confusing to internal customers. I had a request to clarify this, so I added the heading and revised the intro paragraph. I shortened the heading as well.